templates/correct.html

```diff
@@ -353,6 +349,20 @@ <h3>Tölfræði</h3>
       return fmt.replace(/\[bold\]/g, "<b>").replace(/\[~bold\]/g, "</b>");
    }
 
+   function getReferences(references) {
+      // Add links to references if available
+      var combo = " ";
+      $.each(references, function (rix, r) {
```

Comment:
...seems to me that it can be. Then I suggest a simple `if (!references) return "";` at the beginning.

Reply:
It would be very slightly more efficient to have the if before the var. And I am skeptical about the two spaces in the combo string, suggest just leaving one there (we are increasing margin-left from 5px to 8px so that partially compensates).